drv-smoke-tests/smoke_test.rb

```diff
@@ -1,79 +1,101 @@
+    session.assert_text('Summary of title')
+    session.click_link('Find a title')
+    session.assert_text('Search for the title of any property in England and Wales')
+  end
+
+  it 'Can search using the postcode' do
+    visit_title_search_page(session)
+    session.fill_in 'search_term', with: $TEST_TITLE_POSTCODE
+    session.click_button('Search')
+    session.assert_text("#{$TEST_TITLE_NUMBER}")
+  end
+
+  it 'it can navigate to the correct title summary from the postcode search' do
+    visit_title_search_page(session)
+    session.fill_in 'search_term', with: $TEST_TITLE_POSTCODE
+    session.click_button('Search')
+    session.click_link($TEST_TITLE_SEARCH_RESULT_TEXT)
+    session.assert_text("Summary of title #{$TEST_TITLE_NUMBER}")
+  end
+
+  it 'can navigate back to the search page' do
+    visit_title_details_page(session, $TEST_TITLE_NUMBER)
+    session.click_link('Find a title')
+    session.assert_text('Search for the title of any property in England and Wales')
+  end
+
+  it 'can search using an address string' do
+    visit_title_search_page(session)
+    session.fill_in 'search_term', with: $TEST_TITLE_ADDRESS_SEARCH_TERM
+    session.click_button('Search')
+    session.assert_text("#{$TEST_TITLE_NUMBER}")
+  end
+
+  it 'can navigate to the title summary from the address string search' do
+    visit_title_search_page(session)
+    session.fill_in 'search_term', with: $TEST_TITLE_ADDRESS_SEARCH_TERM
+    session.click_button('Search')
+    session.click_link($TEST_TITLE_SEARCH_RESULT_TEXT)
+    session.assert_text("Summary of title #{$TEST_TITLE_NUMBER}")
+  end
 end
-# View a title that has been searched with via postcode
-begin
-  session.click_link($SMOKE_PARTIAL_ADDRESS)
-  session.assert_text("Summary of title #{$SMOKE_TITLE_NUMBER}")
-rescue
-  puts "Unable to navigate to title summary from search page"
+
+def visit_title_search_page(session)
+  session.visit "#{$DIGITAL_REGISTER_URL}title-search"
 end
-# Go back to search page
-begin
-  session.click_link('Find a title')
-  session.assert_text('Search for the title of any property in England and Wales')
-rescue
-  puts "Unable to navigate to search page"
+
+def visit_title_details_page(session, title_number)
+  session.visit "#{$DIGITAL_REGISTER_URL}titles/#{title_number}"
 end
-# Search with address string
-begin
-  session.fill_in 'search_term', with: $SMOKE_PARTIAL_ADDRESS
-  session.click_button('Search')
-  session.assert_text("#{$SMOKE_TITLE_NUMBER}")
-rescue
-  puts "Address string search has failed"
+
+def visit_login_page(session)
+  session.visit "#{$DIGITAL_REGISTER_URL}login"
 end
-# View a title that has been searched with via postcode
-begin
-  session.click_link($SMOKE_PARTIAL_ADDRESS)
-  session.assert_text("Summary of title #{$SMOKE_TITLE_NUMBER}")
-rescue
-  puts "Unable to navigate to title summary from search page"
+
+def visit_landing_page(session)
+  session.visit "#{$DIGITAL_REGISTER_URL}"
 end
```

Comment:
The landing page and the login page are technically the same thing, is there a reason why you have separated them out like this?